@here/harp-datasource-protocol/lib/StringEncodedNumeral.ts

```diff
@@ -153,6 +176,13 @@ export const StringEncodedNumeralFormats: StringEncodedNumeralFormat[] = [
     ...StringEncodedColorFormats
 ];
 
+export const StringEncodedNumeralFormatMaxSize = Math.max(
```

Comment:
Do you think you could find a way to remove these exports?

Reply:
Ok I reduced them to single export, Color and Metric format sizes are not used outside this file scope.
Anyway I could remove them all, but then we may pass any array and expand it dynamically when decoder receives something smaller - it is of course possible in JS:
`
decoder: (encodedValue: string, target: number[]) => {
    if (target.length < this.size) {
       target.length = this.size'
    }
    ...
}
`
Or even simpler, just write the values for certain indexes and JS will grow array automatically, but I am not sure how does this mechanism may affect performance on all platforms.
`
decoder: (encodedValue: string, target: number[]) => {
   ...
    target[0] = r;
    target[1] = g;
    target[2] = b;
    ...
`
Summarizing I am not sure if this is worth effort of hiding one constant. Please decide which approach you would prefer to see.